resources/rules/pre_identifier_merge/step_b_id_exact_match/datasource_xref/eg_gene_info_rules.clj

```diff
@@ -1,10 +1,10 @@
-;; EG gene id to other gene id mappings
+;; --------------------------------
+;; --------- EG Gene Info ---------
+;; --------------------------------
 `{:name "eg-gene-exact-mapping-assertion"
- :head ((?/geneid skos/exactMatch ?/otherice))
-  :body
-  (~@(kabob/rtv _/record
-      iaoeg/EntrezGeneInfoFileData_geneIDDataField1 ?/geneid
-      iaoeg/EntrezGeneInfoFileData_dbXrefsDataField1 ?/otherice)
-   (?/otherice kiao/denotesSubClassOf obo/SO_0000352))} ;dna
-    ;;rdf/type kiao/DNAIdentifier))}
-
+  :description "This rule asserts an exact match between eg genes and other gene ids"
+  :head ((?/geneid skos/exactMatch ?/otherice))
+  :body ((?/record iaoeg/EntrezGeneInfoFileData_geneIDDataField1 ?/geneid)
+         (?/record iaoeg/EntrezGeneInfoFileData_dbXrefsDataField1 ?/otherice)
+         (?/otherice kiao/denotesSubClassOf obo/SO_0000352)) ; SO:dna
+  }
```

Comment:
@bill-baumgartner  Ahh, yes that makes much more sense. There will be more of these to fix spread throughout the pre-identifier merge rules. I will try to catch as many as I can, but let me know if you see any I miss.